apptools/undo/__init__.py

```diff
@@ -10,3 +10,10 @@
 """ Supports undoing and scripting application commands.
     Part of the AppTools project of the Enthought Tool Suite.
 """
+import warnings
+
+warnings.warn(
+    ("apptools.undo is deprecated and will be removed in a future release. The"
+     " functionality is now available via pyface.undo"),
+    DeprecationWarning
+)
```

Comment:
We should have stacklevel so that the warning will show where the import happens. Same for the other deprecation warning.

Reply:
Sorry, because this is at `__init__`, the stack is usually not helpful and the level not trackable. Please ignore this.